grammars/puppet.cson

```diff
@@ -146,7 +146,7 @@
     'include': '#variable'
   }
   {
-    'begin': '(?i)\\b(import|include|contain)\\b\\s*'
+    'begin': '(?!.*=>)\\b(import|include|contain|require)\\s+'
```

Comment:
Can't this lookahead be at the end of the capture?
In addition, is `REQUIRE ::something` allowed?